### ISSUE TITLE
Deactivate surface tracking while defining surfaces in player

### ISSUE THREAD

**Issue by ibrilma** [pupil-labs/pupil#2227]
Hi,

is it possible to deactivate surface tracking in pupil player while defining surfaces? We have quite a lot to track and defining all our surfaces takes super long since pupil player tries to update the surface tracking information for every newly defined surface on the fly. That is, before the surface is edited to span the surface-to-track, the entire video is analyzed. This takes up resources and time. 
In short: Would it be possible to deactivate tracking while defining and editing surfaces, and to start the analysis only after all surfaces have been properly defined? I can do some python coding on my own if I would get a hint where to start.

Best
Ingmar

**Comment by papr** [comment]
Hi,

Currently, it is not possible to change this behavior. As a workaround, I suggest the following procedure:
1. Create a recording that is as short as possible but contains all surfaces (no gaze needed)
2. Open it in Player, wait until the marker detection finished running
3. Define all your surfaces (should be less resource-intensive than in the original recording)
4. Close Player
5. Copy the `surface_definitions*` file(s) to all your recordings
6. Open the new recordings. The surfaces should all be defined.

Would that be a suitable solution to your problem?

**Comment by ibrilma** [comment]
Hi Pablo, thank you for your suggestion, that seems like a good workaround. I'll try that! We could even use such a "template definition" for all our subjects, that's cool and saves a lot of work. Thanks! Again, I love the support at pupil labs, really great. BestIngmar--Dr. Ingmar Brilmayer"Communication Electrified"Department of German Language and Literature IUniversity of CologneAlbertus Magnus Platz50923 Cologne, Germanyphone: +49(0)221 470 89910
-------- Ursprüngliche Nachricht --------Von: Pablo Prietz ***@***.***> Datum: 11.03.22  08:45  (GMT+01:00) An: pupil-labs/pupil ***@***.***> Cc: ibrilma ***@***.***>, Author ***@***.***> Betreff: Re: [pupil-labs/pupil] Deactivate surface tracking while defining
  surfaces in player (Issue #2227) 
Hi,
Currently, it is not possible to change this behavior. As a workaround, I suggest the following procedure:

Create a recording that is as short as possible but contains all surfaces (no gaze needed)
Open it in Player, wait until the marker detection finished running
Define all your surfaces (should be less resource-intensive than in the original recording)
Close Player
Copy the surface_definitions* file(s) to all your recordings
Open the new recordings. The surfaces should all be defined.

Would that be a suitable solution to your problem?

**Comment by papr** [comment]
Happy to help!
